impl/client/src/test/javascript/pentaho/data/filter/isGreater.spec.js

```diff
@@ -0,0 +1,118 @@
+/*!
+ * Copyright 2017 Pentaho Corporation.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define([
+  "pentaho/data/filter/isGreater",
+  "pentaho/type/Context",
+  "pentaho/type/complex",
+  "./propertyUtils"
+], function(isGreaterFactory, Context, complexFactory, propertyUtils) {
+
+  "use strict";
+
+  /* global describe:true, it:true, expect:true, beforeEach:true*/
```

Comment:
Like many times, the reasons are historical. Definitely something we can improve. Because this is now widespread through all pentaho/* testing code, we'd probably better do this cleanup in one shot, within a tech-debt story. Do you agree?